lib/FacebookIdModule.js

```diff
@@ -17,22 +18,98 @@ const https = require('https');
  *  TODO: Update the README with FB's Contract
```

Comment:
It would be great if you could update the README with the FB contract so we all know how that call works.